doc/manual/metaprogramming.rst

```diff
@@ -118,8 +137,22 @@ instead of an :obj:`Expr`:
     julia> typeof(ans)
     Symbol
 
+:obj:`Symbol`\ s can also be created using :func:`symbol`, which takes
+a character or string as its argument:
+
+.. doctest::
+
+    julia> :foo == symbol("foo")
+    true
+
+    julia> symbol('\'')
```

Comment:
The ' ' pairing is a character (as mentioned in the text above), its documented [here](http://docs.julialang.org/en/latest/stdlib/punctuation/) which could be better, but its nothing to do with metaprogramming and its explanation IMO doesn't belong in this section.